### DISCUSSION TITLE
2024.3.2

### DISCUSSION THREAD

**Discussion by AndrewBastin** [hoppscotch/hoppscotch#4024] [Announcements]
This is a point release aimed at fixing 2 important bugs: REST and GraphQL pages not rendering while language is French and Selfhosted Instance mailing having issues.

## What's Changed
* fix(common): rest and graphql pages not being rendered when french is selected as app language by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/4004
* fix: resolved mailer module email issue by @balub in https://github.com/hoppscotch/hoppscotch/pull/4000
* refactor: update to dioc v3 by @AndrewBastin in https://github.com/hoppscotch/hoppscotch/pull/4009
* chore: update placeholder text by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/4023

**Full Changelog**: https://github.com/hoppscotch/hoppscotch/compare/2024.3.1...2024.3.2

<hr /><em>This discussion was created from the release <a href='https://github.com/hoppscotch/hoppscotch/releases/tag/2024.3.2'>2024.3.2</a>.</em>

**Comment by mobeicanyue** [comment]
Is this [release](https://github.com/hoppscotch/releases) an open source build?
why not automatically building it via github action?
